enterprise/backend/test/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions_test.clj

```diff
@@ -271,10 +271,15 @@
                (run-venues-count-query)))))
 
     (testing "Make sure that you can still use a SQL-based GTAP without needing to have SQL read perms for the Database"
-      (is (= [["Red Medicine"] ["Stout Burgers & Beers"]]
-             (mt/rows
-               (mt/with-gtaps {:gtaps {:venues (venue-names-native-gtap-def)}}
-                 (mt/run-mbql-query venues {:limit 2}))))))
+      (let [exp (if (= driver/*driver* :oracle)
+                    [[1M "Red Medicine"]
+                     [2M "Stout Burgers & Beers"]] ;; Oracle returns the IDs as BigDecimal for some reason
+                    [[1 "Red Medicine"]
+                     [2 "Stout Burgers & Beers"]])]
+        (is (= exp
+               (mt/rows
+                 (mt/with-gtaps {:gtaps {:venues (venue-names-native-gtap-def)}}
+                   (mt/run-mbql-query venues {:limit 2, :order-by [[:asc [:field (mt/id :venues :id)]]]})))))))
```

Comment:
Use `mt/formatted-rows [int str]` instead of `mt/rows` and you won't need to add special casing for `:oracle`. It's better not to do that sort of special casing in the expected form if you can avoid it since it adds up quickly when other quirks come in to play